src/BuildScriptGenerator/DefaultRunScriptGenerator.cs

```diff
@@ -0,0 +1,64 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Extensions.Logging;
+using Microsoft.Oryx.BuildScriptGenerator.Exceptions;
+using Microsoft.Oryx.Common;
+
+namespace Microsoft.Oryx.BuildScriptGenerator
+{
+    internal class DefaultRunScriptGenerator : IRunScriptGenerator
+    {
+        private const string TempScriptPath = "/tmp/run.sh";
+
+        private readonly IEnumerable<IProgrammingPlatform> _programmingPlatforms;
+        private readonly ILogger<DefaultRunScriptGenerator> _logger;
+
+        public DefaultRunScriptGenerator(IEnumerable<IProgrammingPlatform> platforms, ILogger<DefaultRunScriptGenerator> logger)
```

Comment:
I'm not seeing changes to the CLI... how this be called?

Reply:
Let's allow the user to specify the output file (like the actual startup gens in the run images) as opposed to just print them in the stdout. That pattern allows us to print logs of what we're doing to the stdout.

Reply:
it's already like that - there's a `--output` option